db/migrate/20191120222312_create_user_and_user_sources.rb

```diff
@@ -0,0 +1,16 @@
+class CreateUserAndUserSources < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users, id: :uuid do |t|
+      t.integer :num_annotation_characters, default: 0
+      t.integer :num_highlights, default: 0
+    end
+
+    create_table :user_sources, id: :uuid do |t|
+      t.references :user, type: :uuid, index: true, foreign_key: true
+      t.string :source_id, null: false
```

Comment:
I just realized this table needs `source_type` too.  An `openstax_page` `UserSource` with `source_id` `foo` is distinct from a record with `foo` but a different `source_type`.

Also this table should have a DB uniqueness constraint on `user_id`, `source_id`, and `source_type`.